atom/browser/atom_resource_dispatcher_host_delegate.cc

```diff
@@ -56,7 +61,8 @@ void HandleExternalProtocolInUI(
 
   GURL escaped_url(net::EscapeExternalHandlerValue(url.spec()));
   auto callback = base::Bind(&OnOpenExternal, escaped_url);
-  permission_helper->RequestOpenExternalPermission(callback, has_user_gesture);
+  permission_helper->RequestOpenExternalPermission(url, has_user_gesture,
```

Comment:
missed a step here of passing the rfh to the permission helper to replace https://github.com/brave/muon/pull/540/files#diff-86fb77c2c8ae419a7c9aeb2f70a84dffR59